webapp/sass/layout/_markdown.scss

```diff
@@ -3,6 +3,13 @@
 .markdown__heading {
     font-weight: 700;
     line-height: 1.5;
+
+    .emoticon {
```

Comment:
Why not apply these rules to all emoticons? It seems like it could be fragile to have them for just headings.